subgraph/subgraph.yaml

```diff
@@ -0,0 +1,71 @@
+specVersion: 0.0.4
```

Comment:
I think this file has to be different for every network, correct? We probably don't want to check it in then and have it be auto generated when running locally/deploying?

Reply:
The new subgraph CLI includes now allows to use a `networks.json` file that will parse the `subgraph.yml` file based on the network name. 

So running `yarn build --network mainnet` will modify the `subgraph.yml` file with the correct parameters. Sadly there are no ways to pass a template file so it rewrites/modifies the YAML file each time.

The docs also show how to use a template file with using handlebars (like we did before). So far I was trying to use their built-in way of supporting multiple networks but that will indeed lead to the main YAML file being rewritten, which can cause confusion (and even delete code maybe).

https://thegraph.com/docs/en/deploying/deploying-a-subgraph-to-hosted/#using-graph-cli

Reply:
ok so let's add to gitignore?

Reply:
the `subgraph.yml` is actually used as template so we can need it checked in. I will see if there is a way to specify a file path to use